components/sysutils/virtualbox/patches/20-ctf.patch

```diff
@@ -0,0 +1,235 @@
+--- VirtualBox-6.1.22/kBuild/tools/GCC3PLAIN.kmk	1970-01-01 00:00:00
```

Comment:
Hi, why do we need this one? If we really want ctf data in VirtualBox (I'm not sure we do), at least remove OOCEVER mentions.

Reply:
i am sorry, i really don't know this stuff. i just copied over the patches and  'make-it-work-for-me'.
please suggest what we/i should do with this..